app/helpers/template_helpers.py

```diff
@@ -1,304 +1,376 @@
+        self._is_post_submission = is_post_submission
+        self._include_csrf_token = include_csrf_token
+        self.context_options = context_options
+        self._sign_out_url = url_for("session.get_sign_out")
+        self._account_service_url = cookie_session.get(
+            "account_service_url", self.context_options.account_service_url
         )
-    else:
-        help_path = (
-            "help/help-with-the-questions/online-questions-help/"
-            if is_nisra_theme
-            else "help/how-to-answer-questions/online-questions-help/"
+        self._account_service_log_out_url = cookie_session.get(
+            "account_service_log_out_url"
         )
-        cookies_path = "cookies/"
-        accessibility_statement_path = "accessibility-statement/"
-        privacy_and_data_protection_path = "privacy-and-data-protection/"
-        terms_and_conditions_path = "terms-and-conditions/"
-        contact_us = "contact-us/"
-        languages = "help/languages-and-accessibility/languages/"
-        bsl_and_audio_videos = (
-            "help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/"
+        self._cookie_settings_url = current_app.config["COOKIE_SETTINGS_URL"]
+        self._cdn_url = (
+            f'{current_app.config["CDN_URL"]}{current_app.config["CDN_ASSETS_PATH"]}'
+        )
+        self._address_lookup_api = current_app.config["ADDRESS_LOOKUP_API_URL"]
+        self._google_tag_manager_id = current_app.config.get("EQ_GOOGLE_TAG_MANAGER_ID")
+        self._google_tag_manager_auth = current_app.config.get(
+            "EQ_GOOGLE_TAG_MANAGER_AUTH"
         )
-    urls = {
-        "help": f"{base_url}{help_path}",
-        "cookies": f"{base_url}{cookies_path}",
-        "accessibility_statement": f"{base_url}{accessibility_statement_path}",
-        "privacy_and_data_protection": f"{base_url}{privacy_and_data_protection_path}",
-        "terms_and_conditions": f"{base_url}{terms_and_conditions_path}",
-        "contact_us": f"{base_url}{contact_us}",
-    }
-    if not is_nisra_theme:
-        urls["languages"] = f"{base_url}{languages}"
-        urls["bsl_and_audio_videos"] = f"{base_url}{bsl_and_audio_videos}"
 
-    return urls
+    @property
+    def context(self) -> dict[str, Any]:
+        return {
+            "account_service_url": self._account_service_url,
+            "account_service_log_out_url": self._account_service_log_out_url,
+            "contact_us_url": Link(
+                lazy_gettext("Contact us"),
+                f"{self.context_options.base_url}/contact-us/",
+            ).__dict__,
+            "cookie_settings_url": self._cookie_settings_url,
+            "page_header": self.page_header_context,
+            "footer": self.footer_context,
+            "languages": get_languages_context(self._language),
+            "theme": self.context_options.design_system_theme,
+            "language_code": self._language,
+            "schema_theme": self._theme,
```

Comment:
I don't think this is used anywhere and can be removed, both the context property and the `_theme` class property.